### DISCUSSION TITLE
Bring Material 3 as Default stype of building apps on Sketchware Pro

### DISCUSSION THREAD

**Discussion by TheRealAshik** [Sketchware-Pro/Sketchware-Pro#1437] [Ideas]
Hey there, can you please update the default `styles.xml` file and other settings so that apps use Material 3 components by default? It's a pain to have to manually replace these components, and it causes issues like the toolbar text staying white even after switching to Material 3. According to the Material 3 design guidelines, the toolbar (or action bar) should have black text. Changing the default settings would not only follow these guidelines but also make it easier for us developers who use Sketchware Pro. Thanks!

---

### Why Material 3 is Beautiful ✨

Material 3, also known as Material You, is Google's latest design system that brings a new level of personalization and aesthetics to your app. It offers dynamic color schemes that adapt to the user’s wallpaper and preferences, creating a more personalized and immersive experience. Here are a few reasons why Material 3 stands out:

- **Personalization**: Material 3 adapts colors and shapes based on user preferences, making each app experience unique.
- **Modern Look**: It features updated components and typography, giving apps a fresh and contemporary appearance.
- **Accessibility**: Enhanced focus on accessibility ensures that apps are usable by everyone, with improved contrast ratios and legibility.
- **Consistency**: Adopting Material 3 ensures a consistent design language across all Android apps, providing a seamless user experience.

By updating your default styles to Material 3, not only will your app look modern and professional, but you'll also save time on manual adjustments and ensure your design aligns with the latest best practices. 🌈

**Comment by trindadedev13** [comment]
What about people who don't like Material 3?

**Reply by TheRealAshik** [reply]
Most of the people likes M3 I Guess.

**Reply by trindadedev13** [reply]
but not all, I know many who don't like it

**Reply by laymintu** [reply]
> Most of the people likes M3 I Guess.

I don't like the M3 design

**Comment by egecanakincioglu** [comment]
Is there any particular reason why you chose Material 3?

**Reply by RofrancoD** [reply]
It would be feasible to just put an option that indicates which style you want m3 or the default so everyone is happy, I do it in a way that let's say manual and certain things, but it would be good to have something more current even if it is something light.

**Comment by TheRealAshik** [comment]
> Is there any particular reason why you chose Material 3?

Up to date UI

**Reply by waelchateur** [reply]
done wait for PR to be merged
now sketchware is m3 ready

**Comment by RoyTheReal** [comment]
YES
